hudi-common/src/main/java/org/apache/hudi/common/model/OverwriteWithLatestAvroPayload.java

```diff
@@ -79,6 +79,11 @@ public OverwriteWithLatestAvroPayload preCombine(OverwriteWithLatestAvroPayload
    * @returns {@code true} if record represents a delete record. {@code false} otherwise.
    */
   protected boolean isDeleteRecord(GenericRecord genericRecord) {
+    final String isDeleteKey = "_hoodie_is_deleted";
+    // Modify to be compatible with old version Avro.
+    if (genericRecord.getSchema().getField(isDeleteKey) == null) {
+      return false;
+    }
     Object deleteMarker = genericRecord.get("_hoodie_is_deleted");
```

Comment:
Why do not change the literal to the `_hoodie_is_deleted ` that you defined?

Reply:
We should fix the logic for Avro version that higher than 1.10, because the higher version throws directly if the field name does not exist in the schema.

Reply:
I mean you have defined the `isDeleteKey`, why do not change the literal in this line?

Reply:
Okey, got your idea, thanks ~